bintray.gradle

```diff
@@ -0,0 +1,26 @@
+//
+// This file is to be applied to some subproject.
+//
+
+apply plugin: 'com.jfrog.bintray'
+
+bintray {
+    user = bintray_user
+    key = bintray_api_key
+    configurations = ['archives']
+    pkg {
+        repo = 'org.jmonkeyengine'
```

Comment:
I don't understand. the 'repo' is not the groupId, 'repo' + 'userOrg' define the destination repository on bintray.